clarify how to declare JavaFX dependency in SDK 6 (#1258)
MicroEJ/docs: VEEPortingGuide/mock.rst

>    .. tab:: SDK 6

Reviewer: "The installation instructions depends on the SDK version:" -> "The installation instructions depend on the SDK version:"

>       - JavaFX must be added as a compile-time dependency in your Mock project.

Reviewer: ```suggestion
      - add JavaFX as a compile-time dependency in your Mock project:
```

Author: Done.

         .. code-block:: kotlin

            compileOnly(group="com.microej.tool", name="javafx", version="1.2.0", configuration="provided")

>       - If your VEE Port contains at least one Mock, JavaFX must be added as a Mock dependency in the VEE Port project.

Reviewer: ```suggestion
      - If your VEE Port contains at least one Mock, add JavaFX as a Mock dependency in your VEE Port project:
```

Author: Done.
